rero_ils/modules/loans/tasks.py

```diff
@@ -34,11 +35,25 @@ def loan_anonymizer(dbcommit=True, reindex=True):
     :param dbcommit: commit record to database.
     :return a count of updated loans.
     """
-    loans_count = 0
-    for org_pid in Organisation.get_all_pids():
-        loans_count_org = anonymize_loans(
-            org_pid=org_pid, dbcommit=dbcommit, reindex=reindex)
-        loans_count = loans_count + loans_count_org
-    msg = f'number_of_loans_anonymized: {loans_count}'
+    query = PatronsSearch() \
+        .filter('bool', must_not=[
+            Q('exists', field='keep_history'),
+            Q('term', keep_history=True)
+        ])
+    anonym_patron_pids = [h.pid for h in query.source('pid').scan()]
+    query = LoansSearch() \
+        .filter('terms', patron_pid=anonym_patron_pids) \
+        .filter('term', to_anonymize=False) \
```

Comment:
Isn't `True` instead of `False` ? You are searching about loan to anonymize candidates.

Reply:
No, we searching for loans not anonymized jet.